shared/src/common/enums/validation/master/master-sign-up.validation-rule.ts

```diff
@@ -0,0 +1,7 @@
+const MasterSignUpValidationRule = {
```

Comment:
rename file to `shared/src/common/enums/validation/master/master-sign-up-validation-rule.enum.ts`